blueprints/app/files/.eslintrc.js

```diff
@@ -2,12 +2,39 @@ module.exports = {
   root: true,
   parserOptions: {
     ecmaVersion: 2017,
-    sourceType: 'module'
+    sourceType: 'module',
   },
   extends: 'eslint:recommended',
   env: {
-    browser: true
+    browser: true,
   },
   rules: {
-  }
+  },
+  overrides: [
+    // node files
+    {
+      files: [
+        './ember-cli-build.js',
+        './testem.js',
+        './index.js',
```

Comment:
should we use `./*.js` instead?

Reply:
Interesting. I hadn’t thought of that. Should work nicely.

Reply:
Sadly, this doesn't work.  The main issue is that in the implementation (https://github.com/eslint/eslint/pull/8081) when the `files` and `excludedFiles` are passed in to `minimatch` ([here](https://github.com/eslint/eslint/blob/8e1a095f4cc3b371a0a424b66e13ec77734193a2/lib/config/config-ops.js#L368-L382)) they also pass `{ matchBase: true }` as options. 

From the `minimatch` README about `matchBase`:

> If set, then patterns without slashes will be matched against the basename of the path if it contains slashes. For example, a?b would match the path /xyz/123/acb, but not /xyz/acb/123.

**tl;dr** Due to the usage of `matchBase` it is not possible to use globs to match files in the project root. If we use `*.js` it matches _all_ files in the repo, if we use `./*.js` it matches no files, if we use `/*.js` it throws an error....

😭 😭 😭